app/views/projects/public_show.html.erb

```diff
@@ -3,6 +3,13 @@
 <% provide(:meta_image, image_url(@project.primary_image.url)) if @project.has_images? %>
 
 <header class="project">
+  <% prev_project, next_project = prev_project_id(@project), next_project_id(@project) %>
+  <% if prev_project || next_project %>
+    <div class="prev-next-links">
+      <%= link_to t('.prev-project'), project_path(prev_project), :class => 'prev-project' if prev_project %>
```

Comment:
We actually need the full project object here since the canonical URL includes a project slug (see https://github.com/awesomefoundation/awesomebits/blob/master/app/models/project.rb#L103-L105)

Reply:
Right now this URL gets redirected to a proper canonical (guess by the projects controller).

I wanted to keep it that way for simplicity.

But ok, let's do a full URL from the beginning.